### ISSUE TITLE
error on app integration

### ISSUE THREAD

**Issue by umer322** [wilburt/flutter_paystack#18]
after integrating this package i'm getting this error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:preDebugBuild'.
> Android dependency 'androidx.core:core' has different version for the compile (1.0.0) and runtime (1.1.0-alpha03) classpath. You should manually set the same version via DependencyResolution

**Comment by wilburt** [comment]
Please see https://github.com/flutter/flutter/issues/27254 for the solution.

**Comment by umer322** [comment]
thanks man

**Comment by umer322** [comment]
when i dont pass access code i get this error.
[ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: Pass an
accesscode

when i put random access code just to check i get this error.
[ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: You can
not set access code when providing transaction parameters in app

**Comment by wilburt** [comment]
The first error was addressed in #17.
The second can be fixed by providing access code or reference before any other value. Something like this:
```
Charge charge = Charge()
      ..reference = _getReference()
       // or ..accessCode = _getAccessCodeFrmInitialization()
      ..email = 'customer@email.com';
```
